launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml

```diff
@@ -148,32 +148,33 @@
     </node>
   </group>
 
+  <!-- V2X fusion -->
+  <group>
+    <include file="$(find-pkg-share traffic_light_arbiter)/launch/traffic_light_arbiter.launch.xml">
+      <arg name="perception_traffic_signals" value="$(var fusion/traffic_signals)"/>
```

Comment:
In the traffic_light_arbiter, since it involves arbitrating between internal and external sources, I would like the topics being received to be explicitly named as "internal" and "external". :pray:

Reply:
Thank you for the review, I have updated the topic name.